google-cloud-spanner/src/main/resources/com/google/cloud/spanner/connection/ClientSideStatements.json

```diff
@@ -474,6 +474,45 @@
 			"allowedValues": "(TRUE|FALSE)",
 			"converterName": "ClientSideStatementValueConverters$BooleanConverter"
 		}
+	},
+	{
+		"name": "SET PROTO_DESCRIPTORS = '<proto_descriptor>'",
+		"executorName": "ClientSideStatementSetExecutor",
+		"resultType": "NO_RESULT",
+		"statementType": "SET_PROTO_DESCRIPTORS",
+		"regex": "(?is)\\A\\s*set\\s+proto_descriptors\\s*(?:=)\\s*(.*)\\z",
+		"method": "statementSetProtoDescriptors",
+		"exampleStatements": ["set proto_descriptors='protodescriptorsbase64'"],
```

Comment:
It surprises me that this works with the generated tests. Will the Base64 decoder not complain about this?

Reply:
In this particular example, `protodescriptorsbase64` has characters that are valid base64 characters. However, if we add some space or any invalid character then the decoder throws an exception.

Reply:
Ah, makes sense. Thanks for clarifying.